app.rb

```diff
@@ -1,12 +1,111 @@
+require_relative 'genre'
+require_relative 'music_album'
+require 'json'
+
 class App
+  def initialize
+    @music_albums = []
+    @genres = []
+  end
+
+  def load_data
+    load_music_albums_data
+    load_genres_data
```

Comment:
I think it would make sense to move these two methods inside `#initialize`. This would remove the need to have a `load_data` method and even remove the need to call `app.load_data` in the main method.